### PR TITLE
Feature/improve requesting feedback 177

### PR DESCRIPTION
**Pull request by TomOffringa**
Wait with merge until backend fixed 177 in API.

### CODE REVIEW

File: src/pages/AskFeedback.js [hunk 1]
```diff
@@ -46,6 +46,11 @@ const renderSelect = ({
           {option.text}
         </option>
       ))}
+      {label === 'circle' && (
+        <option key="none_1337" value="none">
```
Comment (PM5544) [inline src/pages/AskFeedback.js]: key attributes are only needed for JSX elements created inside a loop (via map ors something), without this this becomes a one-liner and the parenthesis are also removed by Prettier.
And can you rename the value of the <option> to "no_circle", I think this adds clarity especially in the render where this value is checked.

File: src/pages/AskFeedback.js [hunk 2]
```diff
@@ -109,8 +114,13 @@ class AskFeedback extends Component {
       persons: []
     });
 
-    this.load({ path: 'roles', circleId: value });
-    this.load({ path: 'person', type: 'persons', circleId: value });
+    if (value === 'none') {
```
Comment (PM5544) [inline src/pages/AskFeedback.js]: change this value to "no_circle"?

**Review by PM5544 — CHANGES_REQUESTED** [review]
Besides these point 👍🏻

**Comment by PM5544** [comment]
awesomesauce

**Comment by TomOffringa** [comment]
Please wait with merging until https://github.com/wearespindle/flindt/pull/72 is merged and deployed.

**Review by PM5544 — APPROVED** [review]

File: src/pages/AskFeedback.js [hunk 3]
```diff
@@ -46,6 +46,9 @@ const renderSelect = ({
           {option.text}
         </option>
       ))}
+      {label === 'circle' && (
+        <option value="no_cirlce">No Specific Circle</option>
```
Comment (PM5544) [inline src/pages/AskFeedback.js]: oopsie a typo :) 'no_cirlce' ==> 'no_circle'

**Review by PM5544 — CHANGES_REQUESTED** [review]

**Review by PM5544 — DISMISSED** [review]

**Pull request closed without merging**